### PR TITLE
[DO NOT MERGE] Requesting final review for Ralsei concept launch (debt)

### PR DESCRIPTION
**Pull request by alicerunsonfedora**
I'm requesting a final review of the changes in the `ralsei` branch before launching it as an open concept demo. I'm checking for the following flags:

- [ ] Code cleanliness
- [ ] Maintains policies set by FCPs on the [policies page](https://theangelreturns.aliceos.app/policies.html)
- [ ] Minor writing revisions
- [ ] AliceOS modifications and additions

## Note
This is **not** to be merged for the time being. This is a duplicate of #35 that offers a better reviewing system. If reviews are successful and we launch this branch as an open-concept demo, the results of the demo's feedback will determine whether the PR gets merged to project Nova Scotia or not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Doki Doki: The Angel Returns
+
 [![Build Status](https://travis-ci.org/ProjectAliceDev/the-angel-returns.svg?branch=nova-scotia)](https://travis-ci.org/ProjectAliceDev/the-angel-returns)
 
-> This is the for the branch `nova-scotia`, dedicated to the developments of Project Nova and Project Scotia. To get started with quick development on Nova/Scotia, you can get started with [your own Gitpod workspace](https://gitpod.io/#https://github.com/ProjectAliceDev/the-angel-returns/tree/nova-scotia) or [use an existing one](https://c53bfdb0-f2e1-45e2-bcb5-cfa29a105200.ws-us.gitpod.io/
+> This is the for the branch `ralsei`, dedicated to the developments of Project Nova and Project Scotia. To get started with quick development on Nova/Scotia, you can get started with [your own Gitpod workspace](https://gitpod.io/#https://github.com/ProjectAliceDev/the-angel-returns/tree/nova-scotia) or [use an existing one](https://c53bfdb0-f2e1-45e2-bcb5-cfa29a105200.ws-us.gitpod.io/
```
Comment (sr229) [inline README.md]: Sharing your existing workspace also shares your credentials, this is highly discouraged. Remove the line that defines your workspace and just use the `gitpod.io#` link instead.

**Review by sr229 — CHANGES_REQUESTED** [review]

**Review by sr229 — APPROVED** [review]

File: travis/inject_version.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # No need to change anything below the replacer variable.
 # Just change the value of replacer, the script will do everything for you
```
Comment (DuskEalain) [inline travis/inject_version.sh]: /// Dusk here - I don't see anything wrong so far.

Comment (sr229) [reply travis/inject_version.sh]: Actually I don't think WE should change that, we set the versioning via Git tags already, what's the point of this?

Comment (alicerunsonfedora) [reply travis/inject_version.sh]: The 'Send Feedback' button will not appear if the word "beta" doesn't appear in `config.version`.

**Comment by alicerunsonfedora** [comment]
I'm just forcing the Send Feedback button to be displayed.

File: travis/inject_version.sh (hunk 2 above)
Comment (sr229) [reply travis/inject_version.sh]: but we override this nonetheless, which doesn't make sense

Comment (sr229) [reply travis/inject_version.sh]: Either suffix all tags with `-beta` then.

Comment (alicerunsonfedora) [reply travis/inject_version.sh]: No need to worry. I just removed the check and made it a permanent button.

**Pull request closed without merging**